contrib/util/language_translations/currentConstants.txt

```diff
@@ -5542,7 +5542,7 @@ Mandatory
 Mandatory and specified fields
 Mandatory or specified fields only, dup check, no search
```

Comment:
also shouldn't be modifying anything in this directory since part of the translation pipeline